invenio_communities/templates/semantic-ui/invenio_communities/details/header.html

```diff
@@ -21,7 +21,7 @@
           <div class="flex rel-mb-1">
             <h2 class="ui header mb-0">{{ community.metadata.title }}</h2>
             {% if community.access.visibility == 'restricted' %}
-              <div class="rel-ml-1">
+              <div class="rel-ml-1 access-status-tag">
```

Comment:
I am not sure whether this is as good approach. 

The problem is that the `access-status` label has two elements: `span` and `icon`. In some cases, they can break line if there is no space for the label to render. To fix this, I set the property `min-width: fit-content`  to enforce that the whole div is always capable of containing both elements.

We can enforce this behavior in every `access-status` label that has an icon, but I think it only breaks in the community header because we're using flex.

Please feel free to suggest an alternative solution.